src/ui/public/doc_table/components/table_row.js

```diff
@@ -51,6 +51,13 @@ module.directive('kbnTableRow', function ($compile, $httpParamSerializer, kbnUrl
       // when we compile the toggle button in the summary, we use this $scope
       let $toggleScope;
 
+      $scope.copyToClipboard = (uriescape) => {
+        let text = window.location.origin + window.location.pathname;
+        text += '#/doc/' + $scope.indexPattern.id + '/' + $scope.row._index + '/' +
```

Comment:
Could you please use template strings instead of concatenation.

Reply:
Done.